client/web/src/dashboard/DashboardComponent.js

```diff
@@ -263,7 +267,7 @@ export const DashboardComponent = (props) => {
                                 </ECRInstructions>
                               </dt>
                               <dd className="mb-3">
-                                {awsAccount}.dkr.ecr.{awsRegion}.amazonaws.com
+                                {awsAccount}.dkr.ecr.{awsRegion}.{serviceUrlSuffix}
```

Comment:
It looks like you've replaced `amazonaws.com` with `amazon.com` in non-GCR regions.

Reply:
It's a mistake, fixed it.